oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/IDManager.java

```diff
@@ -140,6 +159,15 @@ public static String buildRelationId(ServiceInstanceRelationDefine define) {
             return define.sourceId + Const.RELATION_ID_CONNECTOR + define.destId;
         }
 
+        /**
+         * @return encoded instance hierarchy relation id
+         */
+        public static String buildInstanceHierarchyRelationId(InstanceHierarchyRelationDefine define) {
+            return define.instanceId + Const.POINT + define.serviceLayer.value() +
```

Comment:
This is different from `define.serviceId + Const.SERVICE_ID_CONNECTOR + define.serviceLayer.value()`. Could we use a consistent connector? And due to the requirement of https://github.com/apache/skywalking/pull/11698#discussion_r1436427487, a consistent connector should be better for decoding.